independent-publisher-connectors/Wordwide Holidays/apiDefinition.swagger.json

```diff
@@ -0,0 +1,475 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Return public holidays per country code as provided by Nager.Date API",
+    "title": "Worldwide Bank Holidays",
+    "version": "1.0",
+    "contact": {
+      "name": "Reshmee Auckloo",
+      "url": "https://github.com/reshmee011"
+    }
+  },
+  "host": "date.nager.at",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/api/v3/CountryInfo/{countryCode}": {
+      "get": {
+        "tags": [
+          "Country"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "countryCode",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Country Code",
+            "x-ms-url-encoding": "single",
+            "description": "A valid country code such as GB and UA"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CountryInfoDto"
+            }
+          }
+        },
+        "description": "Get country info for the given country",
```

Comment:
@reshmee011 is it possible to give a different description than the summary? please fix all descriptions in the swagger file.